Controller/BaseController.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Razorpay\Magento\Controller;
 
+// require in case of zip installation without composer
+require_once __DIR__ . "/../../Razorpay/Razorpay.php";
```

Comment:
@enishant , this is not required we are adding this only in zip folder file as of now.

Reply:
Removed require_once as we are adding this only in release zip folder file.